documentation/API_documentation/NumberVerification-Readiness-Checklist.md

```diff
@@ -9,14 +9,13 @@ Checklist for number_verification 1.0.0-rc.1 in release r1.1
 |  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |   Y   |      |
 |  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |   Y   |      |
 |  5 | API documentation                            |   M   |         M         |    M    |    M   |   Y   | Embed documentation into API spec - [link](/code/API_definitions/number_verification.yaml)  |
-|  6 | User stories                                 |   O   |         O         |    O    |    M   |   tbd   | [PR #129](https://github.com/camaraproject/NumberVerification/pull/129/files) |
-|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |   tbd   | [PR #124](https://github.com/camaraproject/NumberVerification/pull/124/files) |
-|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |   tbd  | [PR #124](https://github.com/camaraproject/NumberVerification/pull/124/files) |
-|  9 | Test result statement                        |   O   |         O         |    O    |    M   |   tbd   | link |
-
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |   Y   | [get](/documentation/API_documentation/NumberVerification_device_phone_number_User_Story.md) [verify](/documentation/API_documentation/NumberVerification_verify_User_Story.md) |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |   Y   | [PR #124](https://github.com/camaraproject/NumberVerification/pull/124/files) |
+|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |   Y  | [PR #124](https://github.com/camaraproject/NumberVerification/pull/124/files) |

```

Comment:
The status is either "tbd" with the PR, or you consider that PR 124 must be merged before the release PR, than you can already link here the coming .feature file.

Reply:
We have just merged the Test Definition.
I've updated the rc1.2 branch accordingly.